optimum/exporters/tasks.py

```diff
@@ -1078,6 +1097,48 @@ def get_model_from_task(
                 model = model_class.from_pretrained(model_name_or_path, **kwargs)
         return model
 
+    @classmethod
+    def get_task_from_model(
```

Comment:
Apart from using it in `export()`, is there any other use?

Reply:
https://github.com/huggingface/optimum/pull/854/files#diff-4fa3987b008533d0913c0cb0c3dade3fa591f47937b3baa7f41696a17beac945R185

Reply:
Yes, but it was working before so I'm not sure why this change is necessary.

Reply:
It basically does the same thing.

Reply:
Yes, so I'm not sure there's a point adding this method, normally I would think you would get the task from the model id and work from there, not load the model and then try to get the task.